agent.go

```diff
@@ -195,15 +195,6 @@ func (s *sandbox) getProcess(cid, execID string) (*process, *container, error) {
 		return nil, nil, err
 	}
 
-	status, err := ctr.container.Status()
-	if err != nil {
-		return nil, nil, err
-	}
-
-	if status == libcontainer.Stopped {
-		return nil, nil, fmt.Errorf("Container %s is stopped", cid)
-	}
-
 	proc, err := ctr.getProcess(execID)
```

Comment:
It wouldn't hurt to add a comment explaining that this is still a valid call even if the container is stopped.

Also, could you create a unit test for the scenario where the container state is stopped?

Reply:
I'll add the comment and I'll create a simple unit test working for every cases. The function does not need to be tested in particular for a stopped state as this is the point of the patch to allow this function to be called for any container state.